src/Illuminate/Foundation/Console/Tinker/Presenters/IlluminateApplicationPresenter.php

```diff
@@ -0,0 +1,66 @@
+<?php namespace Illuminate\Foundation\Console\Tinker\Presenters;
+
+use Psy\Presenter\ObjectPresenter;
+use Illuminate\Foundation\Application;
+
+class IlluminateApplicationPresenter extends ObjectPresenter {
+
+	protected static $appProps = [
```

Comment:
I think it would be more readable if you just wrote properties rather than props in all cases.